README.md

```diff
@@ -74,14 +67,56 @@ async function up(queryInterface) {
 	});
 }
 
-async function down(queryInterface) {
+async function down({ context: queryInterface }) {
 	await queryInterface.dropTable('users');
 }
 
 module.exports = { up, down };
 ```
 
-See [this test](./test/test.ts) for another example of Umzug usage.
+<details>
+<summary>You can also write your migrations in typescript by using `ts-node`:</summary>
+
+```typescript
+// index.ts
+require('ts-node/register')
+
+import { Sequelize } from 'sequelize';
+import { getUmzug, SequelizeStorage } from 'umzug';
+
+const sequelize = new Sequelize({ dialect: 'sqlite', storage: './db.sqlite' });
+
+const umzug = getUmzug({
+  migrations: { glob: 'migrations/*.ts' },
+  context: sequelize.getQueryInterface(),
+  storage: new SequelizeStorage({ sequelize }),
+});
+
+// export the type helper exposed by umzug, which will have the `context` variable inferred
+export type Migration = typeof umzug._types.migration;
```

Comment:
I think it would be better if we simply provided the Migration type directly to end-users, no? So they can just do `import type { Migration } from 'umzug'` in their migration files, what do you think?

Reply:
We can also do that, but this is a little more specific. `umzug._types.migration` knows about `context` - so in this case the function would have a parameter with a strongly-typed sequelize query interface. If we expose a static interface it won't carry any information about context. But I do accept the `typeof umzug._types.migration` format is awkward. I'll think if there's a better way to get the same result.